spec/unit/installer_spec.rb

```diff
@@ -305,6 +305,28 @@ def @installer.analyze(*)
 
     #-------------------------------------------------------------------------#
 
+    describe '#verify_no_duplicate_library_names' do
```

Comment:
This should either refer to an actual existing method or be put into natural case.